polars/polars-core/src/datatypes/mod.rs

```diff
@@ -82,6 +82,8 @@ impl_polars_datatype!(Int8Type, Int8, i8);
 impl_polars_datatype!(Int16Type, Int16, i16);
 impl_polars_datatype!(Int32Type, Int32, i32);
 impl_polars_datatype!(Int64Type, Int64, i64);
+#[cfg(feature = "dtype-i128")]
+impl_polars_datatype!(Int128Type, Unknown, i128);
```

Comment:
`Unknown` should be `Int128Type`

Reply:
Presumably, you mean `Int128` (which would be `DataType::Int128`)?

If so, I don't think that will work because I think `DataType` will eventually be something like `DataType::Decimal(precision, scale)`, right?

Reply:
Yes, but then I think we should make the `DataType` something like `DataType::Decimal(Option<precision, scale>)`. Then we can fill in the black later.